src/something_game.cpp

```diff
@@ -69,8 +69,8 @@ void Game::handle_event(SDL_Event *event)
     } break;
 
     case SDL_MOUSEMOTION: {
-        mouse_position =
-            camera.to_world(vec_cast<float>(vec2(event->motion.x, event->motion.y)));
+        /* mouse_position = */
+        /*     camera.to_world(vec_cast<float>(vec2(event->motion.x, event->motion.y))); */
```

Comment:
Event object has the most correct position of mouse which takes into account logical size. I think this lines should stay.

Reply:
I'm not sure about two source of true to `mouse_position`.